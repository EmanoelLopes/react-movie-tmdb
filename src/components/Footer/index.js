import React from 'react';

function Footer () {
  return (
    <footer className="rm-footer">
      <div className="container">
        Copyright &copy; 2019
      </div>
    </footer>
  );
}

export default Footer;
