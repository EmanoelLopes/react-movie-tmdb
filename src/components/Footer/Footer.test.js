import React from 'react';
import { shallow } from 'enzyme';
import Footer from 'components/Footer';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<Footer />);
});

afterEach(() => {
  wrapped.unmount();
});

describe('<Footer />', () => {
  it('should match snapshot', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
