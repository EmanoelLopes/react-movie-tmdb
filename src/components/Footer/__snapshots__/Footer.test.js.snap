// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match snapshot 1`] = `
<footer>
   Copyright Â© 2019 
</footer>
`;
